src/index.html

```diff
@@ -8,5 +8,38 @@
   </head>
   <body>
     <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="calendar__day">1</div>
```

Comment:
remove it everywhere
```suggestion
      <div class="calendar__day"></div>
```